app/controllers/catalog_controller.rb

```diff
@@ -206,6 +206,8 @@ def show
     _deprecated_response, @document = search_service.fetch(params[:id])
 
     @cocina = maybe_load_cocina(params[:id])
+    flash[:error] = 'Warning: this object cannot currently be represented in the Cocina model.' if @cocina.instance_of?(NilModel)
```

Comment:
should this be `flash[:warn] = ` as it's a warning, not an error.

Reply:
yes, that's better - i changed it